client/view/base/font-lato.css

```diff
@@ -0,0 +1,27 @@
+@font-face {
+  font-family: 'Lato';
+  font-weight: 400;
+  src: url(./font-lato/Lato-Regular.woff2) format('woff2');
```

Comment:
If you are using file in `base` component, the file should be here.

Another option is to create `index.css` in `view/font-lato` with all `@font-face`.